app/models/bank_account.rb

```diff
@@ -26,15 +26,17 @@ def bank_and_account_name
 
   # rubocop:disable Naming/PredicateName
   def has_tax_credits?
-    true # TODO: CCMS placeholder - to be fixed by AP-1965
+    tax_credits = bank_transactions.pluck(:meta_data).map { |transaction| transaction[:code] }
+    tax_credits.include?('TC') || tax_credits.include?('WTC') || tax_credits.include?('CTC')
```

Comment:
I think we could use a better variable name here - we don't necessarily know that the `meta_data` contains tax credits. 

Something like this might be better:

```suggestion
    meta_data_codes = bank_transactions.pluck(:meta_data).map { |transaction| transaction[:code] }
    meta_data_codes.include?('TC') || meta_data_codes.include?('WTC') || meta_data_codes.include?('CTC')
```

Reply:
Good point - changed